tests/DateTime/ComparisonTest.php

```diff
@@ -41,7 +41,7 @@ public function testGetSetWeekendDays($class)
      */
     public function testEqualToTrue($class)
     {
-        $this->assertTrue($class::createFromDate(2000, 1, 1)->eq($class::createFromDate(2000, 1, 1)));
+        $this->assertTrue($class::createFromDate(2000, 1, 1)->equalTo($class::createFromDate(2000, 1, 1)));
```

Comment:
Shouldn't we keep coverage on the old methods as well?

Reply:
If we plan to duplicate the functionality, certainly. If not, as currently proposed, it wouldn,t hurt.